packages/jetpack-blocks/webpack.config.js

```diff
@@ -0,0 +1,153 @@
+ */
+/* eslint-disable import/no-nodejs-modules */
+
+/**
+ * External dependencies
+ */
+const _ = require( 'lodash' );
+const path = require( 'path' );
+const fs = require( 'fs' );
+const webpack = require( 'webpack' );
+const CopyWebpackPlugin = require( 'copy-webpack-plugin' );
+const DuplicatePackageCheckerPlugin = require( 'duplicate-package-checker-webpack-plugin' );
+const FileConfig = require( '@automattic/calypso-build/webpack/file-loader' );
+const Minify = require( '@automattic/calypso-build/webpack/minify' );
+const SassConfig = require( '@automattic/calypso-build/webpack/sass' );
+const TranspileConfig = require( '@automattic/calypso-build/webpack/transpile' );
+const wordpressExternals = require( '@automattic/calypso-build/webpack/wordpress-externals' );
+
+/**
+ * Internal dependencies
+ */
+const cacheIdentifier = require( '../../server/bundler/babel/babel-loader-cache-identifier' ); // FIX ME
+// const { workerCount } = require( './webpack.common' ); // todo: shard...
+//
+/**
+ * Internal variables
+ */
+const isDevelopment = process.env.NODE_ENV !== 'production';
+const shouldMinify = isDevelopment;
+
+const editorSetup = path.join( __dirname, 'src', 'preset', 'setup', 'editor' );
+const viewSetup = path.join( __dirname, 'src', 'preset', 'setup', 'view' );
+
+function blockScripts( type, inputDir, presetBlocks ) {
+	return presetBlocks
+		.map( block => path.join( inputDir, 'blocks', block, `${ type }.js` ) )
+		.filter( fs.existsSync );
+}
+/**
+ * Return a webpack config object
+ *
+ *
+ * @return {object} webpack config
+ */
+function getWebpackConfig() {
+	const workerCount = 1;
+	const cssFilename = '[name].css';
+	const preserveCssCustomProperties = true;
+
+	const presetPath = path.join( __dirname, 'src', 'preset', 'index.json' );
+	const presetIndex = require( presetPath );
+	const presetBlocks = _.get( presetIndex, [ 'production' ], [] );
+	const presetBetaBlocks = _.get( presetIndex, [ 'beta' ], [] );
+	const allPresetBlocks = [ ...presetBlocks, ...presetBetaBlocks ];
+
+	// Helps split up each block into its own folder view script
+	const viewBlocksScripts = allPresetBlocks.reduce( ( viewBlocks, block ) => {
+		const viewScriptPath = path.join( __dirname, 'src', 'blocks', block, 'view.js' );
+		if ( fs.existsSync( viewScriptPath ) ) {
+			viewBlocks[ block + '/view' ] = [ viewSetup, ...[ viewScriptPath ] ];
+		}
+		return viewBlocks;
+	}, {} );
+	//
+	// Combines all the different blocks into one editor.js script
+	const editorScript = [
+		editorSetup,
+		...blockScripts( 'editor', path.join( __dirname, 'src' ), presetBlocks ),
+	];
+	//
+	// Combines all the different blocks into one editor-beta.js script
+	const editorBetaScript = [
+		editorSetup,
+		...blockScripts( 'editor', path.join( __dirname, 'src' ), allPresetBlocks ),
+	];
+
+	const webpackConfig = {
+		bail: ! isDevelopment,
+		context: __dirname,
+		entry: {
+			editor: editorScript,
+			...( editorBetaScript && { 'editor-beta': editorBetaScript } ),
```

Comment:
We expect to always have an `editor-beta` now:
```suggestion
			'editor-beta': editorBetaScript,
```

Reply:
Yep, we'll always want to build beta files, even when beta list is empty.